### ISSUE TITLE
Add Mute Mic to any Key Binding

### ISSUE THREAD

**Issue by RamseyKal** [seerge/g-helper#3352]
### Rules

- [X] I made myself familiar with the <a href='https://github.com/seerge/g-helper?tab=readme-ov-file'>Readme</a>, <a href='https://github.com/seerge/g-helper/wiki/FAQ'>FAQ</a> and <a href='https://github.com/seerge/g-helper/wiki/Troubleshooting'>Troubleshooting</a>.
- [X] I understand that, if insufficient information will be provided, my issue will be closed without an answer.

### Is your feature request related to a problem? Please describe

I am forced to use M3 Key for brightness down due to M1 and M2 interfering with my Logitech mouse G-Shift, as you know in this issue: #3256 

However, I can't seem to rebind Mute Mic to any other M key or shortcut. Is there a way you can make Mute Mic available in all M keys and all other bindings, please?

### Describe the solution you'd like

Make Mute Mic available to all key bindings.

### Describe alternatives you've considered

I tried searching for a Windows shortcut to control the mute mic on/off but there isn't any.

### Device and Model

Asus ROG Strix G18 G814JZ

**Comment by seerge** [comment]
@RamseyKal hello, try this build
[GHelper.zip](https://github.com/user-attachments/files/17686681/GHelper.zip)

**Comment by RamseyKal** [comment]
Thank you very much, it works great!
Is there a possibility you could add more shortcut to key bindings? Something like FN+X and FN+Z?
I'd love to be able to add a few more commands to those.

**Comment by seerge** [comment]
@RamseyKal ok, great.

As for other hotkeys - only very few FN+<someting> combos emit a keycode, that can be processed. Most do nothing - so you can't bind them.

You can verify it by yourself, as soon as hotkey emits something - app records it to the log (even if it doesn't process it) :)

**Comment by RamseyKal** [comment]
Quick question, is it possible to add a key binding shortcut to switch between 60hz and 240hz if we don't want to use the auto mode? Or that's not a possible Asus command?

**Comment by RamseyKal** [comment]
Oh, nice, but how do I activate F13 on a laptop? I can't use that command. I tried in Custom and entered Ctrl+Shift+Alt+F13.

**Comment by seerge** [comment]
@RamseyKal if you want to rebind macro key to that, check power user settings https://github.com/seerge/g-helper/wiki/Power-user-settings#custom-hotkey-actions

**Comment by RamseyKal** [comment]
Thanks, it works.

For others reading this, this is the command to toggle display refresh rate (I wish it was a setting so that we don't have to keep this code somewhere for later use): 0x11 0xA0 0xA4 0x7C

Is there a way to have the overlay appear when switching? I would love to see, 60Hz, 240Hz, etc., on the screen.

**Comment by seerge** [comment]
@RamseyKal i can add it, but only for hotkey toggle

**Comment by RamseyKal** [comment]
Sure, that would be nice. It's not possible to add it for automated switching I presume, due to Asus limitations, right?

**Comment by RamseyKal** [comment]
In automated switching on my G18, I only see the power mode (silent, balanced) when switching between battery and plugged in states. I don't see a toast notification power source (battery, etc.). Would be nice to see that like with Armoury Crate. And, would be nice to see 60Hz and 240Hz automated. I don't mind seeing all at once.

**Comment by seerge** [comment]
@RamseyKal the icon next to mode name changes depending on power source:)

**Comment by RamseyKal** [comment]
Lol oops, never realised that, thanks for pointing it.

And, is it possible to add this Ctrl+Shift+Alt+F13 (0x11 0xA0 0xA4 0x7C) as an M key binding in G-Helper?

Or here's an idea, perhaps allow us to add them to that list by editing the config. Meaning, if we add certain commands in that file, they would appear in G-Helper's Key Binding list for us to use and we can keep that list for us, if you don't want to add them there. That way, no need to use "Custom" for those. I personally like having all the possible options.

**Comment by seerge** [comment]
@RamseyKal yeah, i have recently allowed to edit binding hotkeys for mode toggling, i can do same for refresh rates and GPU toggling too

**Comment by RamseyKal** [comment]
I meant that we can create our own list and it would appear in Key Binding.
